apps/frontend/src/app/Data/Characters/TravelerHydroF/hydro.tsx

```diff
@@ -326,7 +323,23 @@ export default function dendro(
       }),
     ]),
 
-    passive1: ct.talentTem('passive1'),
+    passive1: ct.talentTem('passive1', [
```

Comment:
why move this?

Reply:
Moved that so that it's easy to understand what the "Healing" refers to, from what it comes. If it's under A1, the mention of Orbs is close by so one will end up reading it and comprehending.